src/Raven.Studio/wwwroot/Content/css/pages/query.less

```diff
@@ -283,6 +283,18 @@
             }
         }
     }
+
+    .tabs-menu {
+        .tooltip-inner {
+            min-width: 100%;
```

Comment:
btw. does it work? As tooltip inner usually is rooted under BODY html tag not here?

Reply:
yes, seems to work, with this the tooltip adjusts itself nicely to the content if the content is **less** than max-width

Reply:
ok - was just double checking